src/components/Header.jsx

```diff
@@ -1,38 +1,43 @@
-// Copyright 2018 Stanford University see LICENSE for license
```

Comment:
This file moved to `src/components/home/Header.jsx` and the file that was in `src/components/editor/Header.jsx` has replaced it.